.gitlab-ci.yml

```diff
@@ -212,6 +215,20 @@ run-tests:
     - job: image-build-tests-runner
     - job: image-build-runner
 
+boardfarm-tests:
+  stage: test
+  image: $CI_REGISTRY_IMAGE/prplmesh-boardfarm-ci:$CI_PIPELINE_ID
+  script:
+    - export USER=''
```

Comment:
Why `Null`-fy `$USER` variable?. Internally, in `PrplMeshDocker` it's used to generate `docker network` ID.

Reply:
Currently prplMesh is built as root, and also has to be run as root with and empty USER value (see https://jira.prplfoundation.org/projects/PPM/issues/PPM-208 and https://jira.prplfoundation.org/projects/PPM/issues/PPM-209).
Because of that we also need to run boardfarm with the same (empty) user.